src/main/java/com/londonkoders/kwitter/user/User.java

```diff
@@ -17,18 +18,19 @@
     @JsonIgnore
     private long id;
 
+    @JsonProperty("name")
```

Comment:
Why do we need `JsonProperty`?

Reply:
I thought the property names in the User class might not be straightforward. Otherwise, we should make a call with a body like this.
{
	"**displayName**" : "Jason",
	"**kwitterHandle**" : "jason_k",
	"bio" : "Hi, I'm Jason!"
}

Do you think we could make it better or keep it as it was?

Reply:
`displayName` was the name we agreed, please see https://github.com/londonkoders/kwitter-service/issues/7#issuecomment-913179145.

We can either rename the field to `name` and use `@JsonProperty("displayName")`, or leave it as-is without the `JsonProperty`.

By the way, it would be great if we can add a Swagger API doc to communicate better with the FE team 🙂

Reply:
you are right! I missed it 😝 I will have a look about Swagger API too. I got rid of `JsonProperty` :)